src/index.css

```diff
@@ -4,5 +4,398 @@
+  height: 19px;
+  font-family: "Roboto Slab", serif;
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.book-option {
+  position: relative;
+  height: 19px;
+  font-family: "Roboto Slab", serif;
+  font-size: 14px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  text-decoration: none;
+}
+
+.middle-option {
+  padding-left: 10px;
+  padding-right: 10px;
+  margin-left: 10px;
+  margin-right: 10px;
+  border-left: solid 1px #e8e8e8;
+  border-right: solid 1px #e8e8e8;
+}
+
+.-Percent-Complete {
+  width: 68px;
+  height: 39px;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 32px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-three);
+}
+
+.Completed {
+  width: 80px;
+  height: 18px;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-three);
+}
+
+.Current-Lesson {
+  width: 75px;
+  height: 21px;
+  font-family: "Roboto Slab", serif;
+  font-size: 16px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: var(--black-three);
+  margin: 8px 0;
+}
+
+.Rectangle-2 {
+  width: 184px;
+  height: 33px;
+  border-radius: 3px;
+  background-color: var(--azure);
+  border: none;
+}
+
+.Update-progress {
+  position: relative;
+  width: 143px;
+  height: 18px;
+  font-family: "Roboto Slab", serif;
+  font-size: 13px;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  text-transform: uppercase;
+  margin: auto;
+}
+
+.progress-numbers {
+  height: 72%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.Rectangle-3 {
+  width: 90px;
+  height: 89.9px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.completion-meter {
+  display: flex;
+  align-items: center;
+}
+
+.Oval-2 {
+  position: relative;
+  width: 68px;
+  height: 68px;
+  background-image: url("./icons/percentage.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.Line-2 {
+  width: 2px;
+  height: 70px;
+  border: solid 1px #e8e8e8;
+}
+
+/*  Form title */
+
+.form-block {
+  margin: 25px auto;
+  width: 1200px;
+  border-top: solid 1px #e8e8e8;
+}
+
+.form-title {
+  height: 24px;
+  margin: 24px 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  text-transform: uppercase;
+}
+
+#inputValue {
```

Comment:
- [ ] [OPTIONAL] 
This is optional in a sense that it's not one of the minimum requirements for this project, but please follow the CSS best practices you learned at the HTML&CSS section.
You should not use ids for styling, they can cause a lot of issues in the cascade as your application grows.

Make sure to change the ids for classes.

[Here's a refresher on why using ids for styling is a bad idea.](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)

Reply:
Thanks for the recommendation. In this particular case, the context made us simply reuse an existing class "in use" by the react application. In general, we haven't defined any new Id only for CSS styling